app/components/markdown/markdown_emoji/__snapshots__/markdown_emoji.test.js.snap

```diff
@@ -0,0 +1,33 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`MarkdownEmoji should match snapshot 1`] = `
+<Unknown
```

Comment:
Not sure why this is `Unknown`.  I supposed it's due to how `commonmark-react-renderer` define its component.  I'm hesitant to add this component but I just added anyway.

Reply:
I think it's because it calls `React.createElement` and passes the methods of the Markdown component as the first argument, so each of these represents a call to something like `Markdown.renderParagraph`. I think this is fine